modules/sor/balancer-sor.integration.test.ts

```diff
@@ -122,26 +130,28 @@ describe('Balancer SOR Integration Tests', () => {
 
     describe('Stable Pool Path', () => {
         beforeAll(async () => {
-            // setup mock pool data
-            const poolAddress = '0x302b75a27e5e157f93c679dd7a25fdfcdbc1473c';
+            // setup mock pool data for a stable pool
+            const poolAddress = '0x1d7699323e1b927c15afe5781fd21a8552a8ea38';
             const stataUSDC = prismaPoolTokenFactory.build({
-                address: '0x8a88124522dbbf1e56352ba3de1d9f78c143751e',
+                address: '0x6bf294b80c7d8dc72dee762af5d01260b756a051',
                 token: { decimals: 6 },
-                balance: '500',
-                priceRate: '1.046992819427282715',
+                balance: '1',
+                priceRate: '1.0',
+
             });
             const stataDAI = prismaPoolTokenFactory.build({
-                address: '0xde46e43f46ff74a23a65ebb0580cbe3dfe684a17',
-                token: { decimals: 18 },
-                balance: '500',
-                priceRate: '1.101882285912091736',
+                address: '0x80d6d3946ed8a1da4e226aa21ccddc32bd127d1a',
+                token: { decimals: 6 },
+                balance: '1',
+                priceRate: '1.0',
```

Comment:
Sorry to be picky about this, but I see you updated the token data but didn't change the variable name. I'm assuming 6 decimals is no longer a `stataDAI` 🙃 - do you mind updating the name?

Reply:
I have changed the variables